package.json

```diff
@@ -11,6 +11,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "body-parser": "^1.19.0",
```

Comment:
It would be nice to have in this file an additional script that will help you to run the client from the root directory. Or even an additional script that will run client + server with the single command (might be useful for development reasons).